spec/services/proceeding_types_service_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe ProceedingTypesService do
+  let!(:legal_aid_application) { create :legal_aid_application, :with_applicant }
+  let!(:proceeding_type) { create :proceeding_type }
+  let!(:default_substantive_scope_limitation) { create :scope_limitation, :substantive_default, joined_proceeding_type: proceeding_type, meaning: 'Default substantive SL' }
+
+  subject { described_class.new(legal_aid_application) }
+
+  describe '#add' do
+    context 'correct params' do
+      let(:scope_limitation) { :substantive }
+      let(:params) do
+        {
+          proceeding_type_id: proceeding_type.id,
+          scope_limitation: scope_limitation
+        }
+      end
+
+      it 'adds a proceeding type' do
```

Comment:
It might be good to have another test checking that when two/multiple proceeding types have been added they are both in `legal_aid_application.proceeding_types`. Maybe something along the lines of 
```
expect { subject }.to change(legal_aid_application.proceeding_types).by(2)
```
in the requests spec maybe by calling patch twice with 2 different ids maybe